From 7156974910b248768f826c75f270a347610174e9 Mon Sep 17 00:00:00 2001
From: Jin Li <jinl@codeaurora.org>
Date: Thu, 19 May 2016 19:18:04 -0400
Subject: [PATCH 36/44] ARM: dts: add hdmi support for drm/kms in 8996 dragon
 board

Adding HDMI properties for DRM/KMS node in 8996 Dragon board to
support KMS features.

Change-Id: I39a9be02781741121fa54a1884eb8fff5bc5c5c4
Signed-off-by: Jin Li <jinl@codeaurora.org>
---
 arch/arm/boot/dts/qcom/apq8096-auto-dragonboard.dtsi | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/arch/arm/boot/dts/qcom/apq8096-auto-dragonboard.dtsi b/arch/arm/boot/dts/qcom/apq8096-auto-dragonboard.dtsi
index 82f19c0..2ca39ec 100644
--- a/arch/arm/boot/dts/qcom/apq8096-auto-dragonboard.dtsi
+++ b/arch/arm/boot/dts/qcom/apq8096-auto-dragonboard.dtsi
@@ -339,6 +339,17 @@
 
 &mdss_mdp {
 	qcom,mdss-pref-prim-intf = "dsi";
+	connectors = <&mdss_hdmi>;
+};
+
+&mdss_hdmi {
+	status = "ok";
+	qcom,hdmi-tx-ddc-clk-gpio = <&tlmm 32 0>;
+	qcom,hdmi-tx-ddc-data-gpio = <&tlmm 33 0>;
+	qcom,hdmi-tx-hpd-gpio = <&tlmm 34 0>;
+	qcom,hdmi-tx-mux-en-gpio = <&tlmm 27 0>;
+	qcom,hdmi-tx-mux-sel-gpio = <&tlmm 83 0>;
+	qcom,hdmi-tx-mux-lpm-gpio = <&tlmm 85 0>;
 };
 
 &mdss_dsi {
-- 
2.8.2

